logic/service/offline-service.js

```diff
@@ -573,23 +567,17 @@ exports.OfflineService = OfflineService = RawDataService.specialize(/** @lends O
     },
 
     readOfflineOperations: {
-        value: function (operationMapToService) {
-            var operationTable,
-                self = this;
-
+        value: function (/* operationMapToService */) {
+            var operationTable = this.operationTable,
+                myDB = this._db;
             return new Promise(function (resolve, reject) {
-                var myDB = self._db;
-
-                myDB
-                .open()
-                .then(function (db) {
-                    self.operationTable.where("operation").anyOf("create","update","delete")
-                    .toArray(function (offlineOperations) {
-                            resolve(offlineOperations);
-                    }).catch(function(e) {
-                            console.log(selector.type + ": performOfflineSelectorChanges failed",e);
-                            console.error(e);
-                            reject(e);
+                myDB.open().then(function (/* db */) {
+                    operationTable.where("operation").anyOf("create", "update", "delete").toArray(function (offlineOperations) {
```

Comment:
I think it was just confused on your intent because you had a variable called operationTable that was not used.  So instead of removing it I just added the myDB property and then you no longer needed self.